packages/web/app/src/config/supertokens/backend.ts

```diff
@@ -441,11 +445,12 @@ async function trySignIntoAuth0WithUserCredentialsAndRetrieveUserInfo(
     headers: { authorization: `Bearer ${accessToken}` },
   });
 
+  const userBody = await userResponse.text();
```

Comment:
why to move it up?

Reply:
wanted to log it, but then I felt lazy, evening is approaching, you know?

Reply:
I understand you well, merge